### ISSUE TITLE
asset-manifest.json is not created in development mode

### ISSUE THREAD

**Issue by vzeleniuk** [micro-frontends-demo/container#27]
asset-manifest.json is not generated in development mode - it will be available after a build was created in production. 
Thus, if the application is running locally, asset-manifest.json is not available and [fetch](https://github.com/micro-frontends-demo/container/blob/ccbb83c8d59a81f6939fbf1aad59f21ba1f8c14e/src/MicroFrontend.js#L13) fails. 

I made the app working by adding this logic for development mode:
```
    const script = document.createElement('script');
    script.id = scriptId;
    script.crossOrigin = '';
    script.src = `${host.replace('http', 'https')}/static/js/bundle.js`;

    script.onload = this.renderMicroFrontend;
    document.head.appendChild(script);
```
Also, I had to handle the protocol issue `host.replace('http', 'https')`.

To summarize, this issue might help someone resolve the same issues that I had or the code owners might consider this and update the application.

**Comment by chrisVillanueva** [comment]
@vzeleniuk Thanks for sharing.  Maybe submit a PR?

**Comment by camjackson** [comment]
Hi @vzeleniuk, I am unable to reproduce this. Here is what I did:

1. `cd` into the directory where the `browse` app is located
2. `yarn start` - the app starts on port 3001
3. Then in another terminal: `curl localhost:3001/asset-manifest.json`

Output:
```json
{
  "main.js": "/static/js/bundle.js",
  "main.js.map": "/static/js/bundle.js.map",
  "index.html": "/index.html"
}
```

So it looks like it's working to me. I'm going to close this for now but if you can tell me how to reproduce it I will reopen.
